api/src/propagation/TextMapPropagator.ts

```diff
@@ -119,12 +119,12 @@ export interface TextMapGetter<Carrier = any> {
 /**
  * @since 1.0.0
  */
-export const defaultTextMapGetter: TextMapGetter = {
+export const defaultTextMapGetter: TextMapGetter<object | null | undefined> = {
```

Comment:
The type changes here is also somewhat of a breaking change, though, it probably wouldn't have worked in practice if you tried to use it with anything else.